### ISSUE TITLE
[Tune] ExperimentAnalysis stats() and runner_data() broken

### ISSUE THREAD

**Issue by bfortuner** [ray-project/ray#5378]
ea = ExperimentAnalysis(path_to_state_json)
ea.stats() or ea.runner_data()
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-58-9b5199f80cfd> in <module>
----> 1 a.runner_data()

~/workplace/dev_cruisetune/.venvpy3/lib/python3.6/site-packages/ray/tune/analysis/experiment_analysis.py in runner_data(self)
    186     def runner_data(self):
    187         """Returns a dictionary of the TrialRunner data."""
--> 188         return self._experiment_state.get("runner_data")
    189 
    190     def _get_trial_paths(self):

AttributeError: 'ExperimentAnalysis' object has no attribute '_experiment_state'
```

Also it looks like trial "status" is no longer returned in the ExperimentAnalysis dataframe, which makes it hard to know which trials succeeded.

**Comment by richardliaw** [comment]
This should be fixed on master (#5365). Try the latest version of master?

**Comment by richardliaw** [comment]
I think this should be fixed and tested with #5365.
